cmake/Modules/FindTestCompileOnly.cmake

```diff
@@ -0,0 +1,4 @@
+#only used in testing for validating compile-flag only libs work
+KOKKOS_CREATE_IMPORTED_TPL(testFlags COMPILE_OPTIONS -ffakeflag)
+KOKKOS_EXPORT_IMPORTED_TPL(Kokkos::testflags)
```

Comment:
I guess we could just get rid of these files before merging.

Reply:
These are removed in the latest commit.